### DISCUSSION TITLE
第4节课笔记(20班)

### DISCUSSION THREAD

**Discussion by Farewell-CK** [InternLM/Tutorial#389] [笔记]
20班的宝子们，第四节课笔记交这里哦

**Comment by deyouzuishuai** [comment]
https://zhuanlan.zhihu.com/p/679021286

**Comment by liu673** [comment]
https://github.com/liu673/InternLM_OpenNotebook/blob/master/lesson_4/lesson_4_note.md

**Comment by 20300240011** [comment]
https://zhuanlan.zhihu.com/p/679658163

**Comment by LiyanJin** [comment]
https://blog.csdn.net/weixin_53443275/article/details/135857581
